services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/Page.java

```diff
@@ -33,30 +36,47 @@
 public final class Page<T> {
     private final List<T> items;
     private final Map<String, AttributeValue> lastEvaluatedKey;
+    private final Integer count;
+    private final Integer scannedCount;
+    private final ConsumedCapacity consumedCapacity;
```

Comment:
I guess we can't just include a service response here since it could be QueryResponse and ScanResponse. :(

Reply:
If we really wanted to, we could make it a metadata field, then we'd get
`page.responseMetadata().count()` instead of
`page.count()`. 

I find the current version a more natural, but if DDB would add many more metadata fields we'd be glad we had a containing object. WDYT?

Reply:
I was thinking about including the whole response`scanResponse`, so `page.sdkResponse().count()`, but that doesn't seem to be possible.